hlink/linking/core/pipeline.py

```diff
@@ -193,6 +193,8 @@ def _calc_categorical_features(
                 "categorical", False
             ):
                 categorical_pipeline_features.append(pipeline_feature["output_column"])
+    logging.info(f"Categorical Comparison features: {categorical_pipeline_features}")
```

Comment:
Typo here, should be `categorical_comparison_features` instead of `categorical_pipeline_features`.

Reply:
Made the change